meta-genivi-dev/recipes-extended/chromium/chromium-wayland_%.bbappend

```diff
@@ -5,6 +5,9 @@ SRC_URI += "\
     file://chromium.service \
 "
 
+# FIXME: workaround for https://at.projects.genivi.org/jira/browse/LM-2
+CHROMIUM_EXTRA_ARGS_append = " --window-size=1728,1080"
+
```

Comment:
What options do we have to try a multi-monitor setup, only R-Car boards? I definitely want to try this out. Also, how does the HMI behave in that case?